lib/openshift/machine.rb

```diff
@@ -26,6 +26,16 @@ def phase(user: nil, cached: true, quiet: false)
         dig('status','phase')
     end
 
+    def instance_state(user: nil, cached: true, quiet: false)
+      raw_resource(user: user, cached: cached, quiet: quiet).
+          dig('status', 'providerStatus', 'instanceState')
+    end
+
+    def annotation_instance_state(user: nil, cached: true, quiet: false)
+      raw_resource(user: user, cached: cached, quiet: quiet).
+          dig('metadata', 'annotations', 'machine.openshift.io/instance-state')
+    end
```

Comment:
@pruan-rht , we already have a method toread annotations in `resource.rb`. Could you remove this redundant method?